Makefile.defs

```diff
@@ -601,7 +601,7 @@ ifdef APPIMAGE
 	HOST_ARCH:=-mtune=generic -march=x86-64
 endif
 
-ifdef LINUX
+ifeq ($(TARGET), linux)
```

Comment:
I assume this will also affect the emulator, which isn't necessarily the biggest deal but if it's not too complicated I'd prefer to avoid that.

Reply:
Why do you mean?

Reply:
Never mind, I assume TARGET is empty for the emulator. I hadn't thought it through. What I mean is the emulator would like to have -march=native (not as a requirement, merely preferentially).

Reply:
> Never mind, I assume TARGET is empty for the emulator.

Yes, we're still compiling the emulator with `-march=native`.

Reply:
And yes, `linux` is not a great target name, considering most of our targets are Linux targets. :P